apps/galaxy/src/modules/auth/SignInForm/SignInForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { ErrorMessage } from "@hookform/error-message";
+import useSignIn from "../../../services/signin/useSignIn";
+import { Button, Container, FormLabel, FormControl, Input } from "@cc/ui-chakra";
+
+interface FormData {
```

Comment:
Maybe name it `LoginFormData` instead? `FormData` feels a bit general.

Reply:
I don't think so, since this form is already in the SignInForm document which has only one form.

Reply:
Okay, that make sense too, happy to leave it as it is.